<launch>
  <node pkg="controller" type="controller" name="control_node" output = "screen"  >
      <param name="control/thresh"   value="0.01" />
      <param name="control/iter_max"       value="200"/>
      <param name="control/car_len"       value="0.5"/>
      <param name="control/traj_order"       value="5"/>
      <param name="control/min_order"       value="2.5"/>
      <remap from="odom" to="odometry"/>
      <remap from="cmd_vel" to="cmd/vel"/>
      <remap from="traj2ctrl" to="traj2ctrl"/>
  </node> 
</launch>